### ISSUE TITLE
Opacity?

### ISSUE THREAD

**Issue by jeetdholakia** [dntks/easyLUT#7]
Can we set the opacity? I'm not sure from 3DLUT docs from android, but the ScriptIntrinsicLUT do have an alpha parameter...

**Comment by dntks** [comment]
Opacity of the LUT image? You can edit it before importing to the project if want, otherwise there is no possibility right now. 
So after adding the filter you want to end up having a semi-transparent image, right?

**Comment by jeetdholakia** [comment]
Yes, I meant the opacity of the LUT image. The use-case is to set a filter strength that we see in so many apps. 
 I've tried using a PNG LUT by setting 50% opacity using photoshop, but it ends up putting more "White color" on the image instead, maybe Android SDK disregards the alpha channel, flattens it like a JPEG? I'm not sure. Any ideas?
